### DISCUSSION TITLE
[12일차-지원] URL와 URI의 차이?

### DISCUSSION THREAD

**Discussion by smarfy99** [zzangCS/zzangInterview#40] [프론트 면접 질문]
URL와 URI의 차이?

**Comment by smarfy99** [comment]
URI(Uniform Resource Identifier)
식별자로 인터넷 상의 리소스 자원 자체를 식별하는 고유 문자열이다.

URL(Uniform Resource Locator)
리소스의 위치를 나타내기 위한 규약. https://가 붙은 것은 URI와 더불어 어떻게 도달할 수 있는지 위치와 프로토콜까지 나타내기 떄문

URL은 일종의 URI이다.

**Reply by smarfy99** [reply]
![image](https://user-images.githubusercontent.com/82459123/234278318-0408f1ca-a255-4190-84d7-fd93675077fe.png)

**Comment by skyhanull** [comment]
Uniform Resource Identifier의 약자인 URI 뜻은 우리말로 ‘통합 자원 식별자’입니다.즉, URI는 인터넷상의 리소스 “자원 자체”를 식별하는 고유한 문자열 시퀀스입니다. 
URL은 Uniform Resource Locator, 네트워크상에서 통합 자원(리소스)의 “위치”를 나타내기 위한 규약입니다

**Comment by ez1n** [comment]
**URI** : Uniform Resource Identifier의 약자로 인터넷상의 리소스 자원 자체를 식별하는 고유한 문자열 시퀀스 (통합 자원 식별자)
**URL** : 네트워크상에서 통합 자원(리소스)의 위치를 나타내기 위한 규약 (웹 주소)

**차이점**
1. URL은 일종의 URI (URI가 더 포괄적인 개념)
2. URL은 자원과 프로토콜이 결합한 형태
